vlm/src/main/scala/RasterSourceRDD.scala

```diff
@@ -59,18 +59,17 @@ object RasterSourceRDD {
     val layerMetadata =
       TileLayerMetadata[SpatialKey](cellType, layout, combinedExtents, crs, layerKeyBounds)
 
-    val sourcesRDD: RDD[(RasterSource, Array[Extent])] =
+    val sourcesRDD: RDD[(RasterSource, Array[SpatialKey])] =
```

Comment:
This is probably the best way to go, as I found that passing either `Extent`s or `GridBounds` can result in weird floating point issues.

Reply:
Saw those as well. It's a little concerning because working from `Extent` should still be a first-class operation. But here working from the pixel/key bounds is a lot more direct anyway.